aws/rust-runtime/aws-config/src/ecs.rs

```diff
@@ -131,6 +131,7 @@ impl ProvideCredentials for EcsCredentialsProvider {
 
 /// Inner Provider that can record failed configuration state
 #[derive(Debug)]
+#[allow(clippy::large_enum_variant)]
```

Comment:
How big is it?

Reply:
432 bytes, the next largest is 56 bytes. Clippy recommends boxing it.